third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/call_acquires_invalid.move

```diff
@@ -0,0 +1,71 @@
+module 0x8675309::M {
```

Comment:
Missing the .exp file?

Reply:
I think this will be fixed after this PR is landed: https://github.com/aptos-labs/aptos-core/pull/12757